src/modules/job-manager/plugins/perilog.c

```diff
@@ -327,6 +322,21 @@ static int run_cb (flux_plugin_t *p,
                    flux_plugin_arg_t *args,
                    void *arg)
 {
+
+    /*  If an epilog command is registered, then we need to subscribe
+     *   to all events for this job to catch job.event.finish.
+     */
+    if (perilog_config.epilog_cmd != NULL) {
+        if (flux_jobtap_job_subscribe (p, FLUX_JOBTAP_CURRENT_JOB) < 0) {
+            flux_jobtap_raise_exception (p,
+                                         FLUX_JOBTAP_CURRENT_JOB,
+                                         "prolog",
```

Comment:
"epilog" instead of "prolog"?

Reply:
oh wait, b/c of the other comment I made, if this was a "fix in a wrong commit", then "prolog" might be fine here.